### ISSUE TITLE
Links should have hover effect

### ISSUE THREAD

**Issue by rain2o** [Myzwer/foothillschurch#29]
https://github.com/Myzwer/foothillschurch/blob/d75494784caf988edbcdb95afebe7becdca009e7/components/cards/message-card.php#L40-L46

Links and buttons should have some sort of visual effect on hover to indicate to the user that they perform an action on click. This also is necessary for a11y standards.

**Comment by Myzwer** [comment]
There may be a few still running around that I'll squash as I go, but this should be the bulk of them.
